demo-notebooks/batch-job/batch_mnist_ray.ipynb

```diff
@@ -63,6 +63,7 @@
     "    min_memory=16,\n",
     "    max_memory=16,\n",
     "    gpu=4,\n",
+    "    image=\"rayproject/ray:2.5.0-py38-cu116\",\n",
```

Comment:
We have a ray image on quay now. We should use that instead.
[quay.io/project-codeflare/ray:2.5.0-py38-cu116](http://quay.io/project-codeflare/ray:2.5.0-py38-cu116)

Reply:
Thanks @anishasthana image name updated